gcc/rust/checks/errors/rust-feature-gate.cc

```diff
@@ -155,4 +155,13 @@ FeatureGate::visit (AST::Function &function)
   check_rustc_attri (function.get_outer_attrs ());
 }
 
+void
+FeatureGate::visit (AST::ExternalTypeItem &item)
+{
+  // TODO(mxlol233): The gating needs a complete visiting chain to activate
+  // `AST::ExternalTypeItem`.
```

Comment:
What do you mean by this comment? `ExternalTypeItem` nodes will be kept as items as part of an `ExternBlock`, which yeah should have all of its items visited. Is that what you mean?

Reply:
> This is nice but missing tests, and I assume part of the visitor for `AST::ExternBlock`? Thank you for working on this :D

Good point, I added a `accept_vis` in   `ExternBlock` visit. And i will add a test case after the pr https://github.com/Rust-GCC/gccrs/pull/1932 is merged, which can make the parsing stage pass. :)

Reply:
> What do you mean by this comment? `ExternalTypeItem` nodes will be kept as items as part of an `ExternBlock`, which yeah should have all of its items visited. Is that what you mean?

The test case is added, plz check :)